packages/web-components/src/tabs/tabs.styles.ts

```diff
@@ -0,0 +1,250 @@
+  }
+  ::slotted([slot='start']) {
+    margin-inline-end: 11px;
+  }
+  ::slotted([slot='end']) {
+    margin-inline-start: 11px;
+  }
+
+  .tabpanel {
+    grid-row: 2;
+    grid-column-start: 1;
+    grid-column-end: 4;
+    position: relative;
+  }
+  :host([orientation='vertical']) {
+    grid-template-rows: auto 1fr auto;
+    grid-template-columns: auto 1fr;
+  }
+  :host([orientation='vertical']) .tablist {
+    grid-row-start: 2;
+    grid-row-end: 2;
+    display: grid;
+    grid-template-rows: auto;
+    grid-template-columns: auto 1fr;
+    position: relative;
+    width: max-content;
+    justify-self: end;
+    align-self: flex-start;
+    width: 100%;
+  }
+  :host([orientation='vertical']) .tabpanel {
+    grid-column: 2;
+    grid-row-start: 1;
+    grid-row-end: 4;
+  }
+  :host([orientation='vertical']) ::slotted([slot='end']) {
+    grid-row: 3;
+  }
+
+  :host([appearance='subtle']) ::slotted(fluent-tab:hover) {
+    background-color: ${colorSubtleBackgroundHover};
+    color: ${colorNeutralForeground1Hover};
+  }
+
+  :host([appearance='subtle']) ::slotted(fluent-tab:active) {
+    background-color: ${colorSubtleBackgroundPressed};
+    fill: ${colorSubtleBackgroundPressed};
+    color: ${colorNeutralForeground1};
+  }
+
+  :host ::slotted(fluent-tab.animated)::after {
+    transition-property: transform;
+    transition-duration: ${durationSlow};
+    transition-timing-function: ${curveDecelerateMax};
+  }
+
+  :host([size='small']) ::slotted(fluent-tab) {
+    font-size: ${fontSizeBase300};
+    line-height: ${lineHeightBase300};
+    padding-top: ${spacingVerticalSNudge};
+    padding-bottom: ${spacingVerticalSNudge};
+    padding-left: ${spacingHorizontalSNudge};
+    padding-right: ${spacingHorizontalSNudge};
+    border-radius: ${borderRadiusMedium};
+  }
+
+  :host([size='large']) ::slotted(fluent-tab) {
+    font-size: ${fontSizeBase400};
+    line-height: ${lineHeightBase400};
+    padding-top: ${spacingVerticalL};
+    padding-bottom: ${spacingVerticalL};
+    padding-left: ${spacingHorizontalMNudge};
+    padding-right: ${spacingHorizontalMNudge};
+    border-radius: ${borderRadiusMedium};
+  }
+
+  /* ::after adds the active indicator  */
+  :host([orientation='horizontal']) ::slotted(fluent-tab)::after {
+    height: ${strokeWidthThicker};
+    margin-top: auto;
+    transform-origin: left;
+    transform: translateX(var(${TabTokenNames.tabIndicatorOffset})) scaleX(var(${TabTokenNames.tabIndicatorScale}));
+  }
+  /* ::before adds a secondary indicator placeholder that appears right after click on the active tab */
+  :host ::slotted(fluent-tab)::before {
+    height: ${strokeWidthThicker};
+    border-radius: ${borderRadiusCircular};
+    content: '';
+    inset: 0;
+    position: absolute;
+    margin-top: auto;
+  }
+
+  :host([orientation='horizontal']) ::slotted(fluent-tab[aria-selected='false']:hover)::after {
+    height: ${strokeWidthThicker};
+    margin-top: auto;
+    transform-origin: left;
```

Comment:
what about rtl here?

Reply:
Just make sure it works in that writing mode

Reply:
when document direction is set to `rtl`

Reply:
ah, okay, this won't be a problem here, since this only targets the hover indicator

Reply:
okay, checked. it's working in ltr and rtl